Automattic/wp-calypso
# Rename new flow to "migration"

Related to #93328

## Proposed Changes

* Repurpose the `MIGRATION_FLOW` constant. The ones using that previously now point to `IMPORT_FOCUSED_FLOW`, which had the same value.
  * Note: It seems the `import-focused` flow is not in use anymore. See paYKcK-4BA-p2.
* Rename the new flow from "hosted-site-migration-v2" to "migration".

## Why are these changes being made?


* The new flow was renamed to a generic name since the plan is to use this flow everywhere.

## Testing Instructions



* Navigate to `/setup/migration`, and make sure the flow works properly.
* Navigate to `/setup/import-focused`, and make sure it also continues loading.

## Pre-merge Checklist
